src/components/Results/Results.tsx

```diff
@@ -33,21 +35,81 @@ const Results: React.FC<Props> = ({ measureFile, patientFile, htmls }) => {
   const results = useRecoilValue(resultsState);
   const detectedIssues = fhirpath.evaluate(results, 'Bundle.entry.resource.DetectedIssue');
 
-  return (
-    <Grid container xs={12} direction="column" spacing={2}>
-      <h2>Results:</h2>
-      <Grid container item xs={12} direction="row" justify="center" alignItems="center">
-        {outputType === 'measureReports' &&
-          calculationOptions.reportType === 'individual' &&
-          fhirpath.evaluate(results, 'MeasureReport.group').map((group: R4.IMeasureReport_Group) => {
-            const id = results ? fhirpath.evaluate(results, 'MeasureReport.subject.reference')[0].split('/') : '';
+  const style = {
+    minWidth: '25%'
+  };
+
+  const [value, setValue] = React.useState('0');
+
+  const handleChange = (event: React.ChangeEvent<{}>, newValue: string) => {
+    setValue(newValue);
+  };
+
+  const displayJSONResults = () => {
+    return (
+      <>
+        <Grid container item xs direction="row">
+          {results && <h2>JSON:</h2>}
+          {results && (
+            <IconButton
+              onClick={() => {
+                fileDownload(
+                  JSON.stringify(results, null, 2),
+                  measureFile.name?.includes('.json')
+                    ? `results-${measureFile.name}`
+                    : `results-${measureFile.name}.json`
+                );
+              }}
+            >
+              <GetApp fontSize="small" />
+            </IconButton>
+          )}
+        </Grid>
+        {results && <ReactJson src={results} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />}
+        {patientFile && <h2>Patient Bundle:</h2>}
+        {patientFile && (
+          <ReactJson src={patientFile} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />
+        )}
```

Comment:
It might be worth looking into if we can center the JSON on the page since there's more real estate now